types/block.go

```diff
@@ -13,6 +13,14 @@ import (
 	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+const (
+	// MaxHeaderBytes is a maximum header size (including amino overhead).
+	MaxHeaderBytes = 476
+
+	// AminoOverheadForBlock - amino overhead to encode the block (whole block + data field).
+	AminoOverheadForBlock = 5
```

Comment:
do we need a script to get these values?

Reply:
At the very least a test should fail if the Header changes and MaxHeaderBytes doesn't change with it.